mainapp/forms.py

```diff
@@ -41,10 +32,24 @@ class Meta:
 class ProfileForm(forms.ModelForm):
     name = forms.CharField(widget=forms.TextInput(attrs={
         'class': 'form-control', 'placeholder': 'name'}), max_length=100)
+    first_name = forms.CharField(widget=forms.TextInput(attrs={
+        'class': 'form-control', 'placeholder': 'First Name'}), max_length=32)
+    last_name = forms.CharField(widget=forms.TextInput(attrs={
+        'class': 'form-control', 'placeholder': 'Last Name'}), max_length=32)
+    email = forms.EmailField(widget=forms.EmailInput(attrs={
+        'class': 'form-control', 'placeholder': 'Enter a valid email'}), max_length=64)
     phone = forms.CharField(widget=forms.TextInput(attrs={
```

Comment:
See [django-phonenumber-field](https://github.com/stefanfoulis/django-phonenumber-field). It handle most common issues (including validation) for phone numbers handling.

Reply:
changed

Reply:
I don't see this change at current file (but I see new migratinon). Check this again ;)